editor/src/clj/editor/app_view.clj

```diff
@@ -1680,7 +1680,7 @@ If you do not specifically require different script states, consider changing th
            (project/sub-select project-id active-resource-node sub-selection open-resource-nodes)))))))
 
 (defn- make-title
-  ([] "Defold Editor 2.0")
+  ([] (str "Defold " (System/getProperty "defold.version" "(No version)")))
```

Comment:
Let's just have it say "Defold" if there is no version. Also, we can use the `(system/defold-version)` function.

```suggestion
  ([] (if-some [version (system/defold-version)]
        (str "Defold " version)
        "Defold"))
```